content/api/servo-assembly/arduino-commands.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Custom Arduino Commands
+---
+
+At the core of the servo assembly is an Arduino <TODO: model name>. By default
+the Arduino runs firmware created by SourceBots providing simple access to the
+attached servo shield, compatible ultrasound sensors and to the GPIO pins
+directly.
+
+You can also extend the firmware on the Arduino to support more features, for
+example if you have other types of sensor which require higher precision timing
+than can be achieved through the Python API.
+
+## Requirements
+
+As your new firmware will still need to be able to talk to the Python code
+running on the Raspberry Pi, you'll need to base it on SourceBots default
+firmware. You can get a copy of the source code from GitHub:
+
+```none
+git clone https://github.com/sourcebots/servo-firmware
+```
+
+<TODO: describe how to do that without git>
```

Comment:
Can we use Go templates comment syntax rather than this or the html comments below.

`{{/* a comment */}}`

Reply:
Aha, thanks. I searched hugo's docs for what the comment syntax was, but couldn't find it!

Reply:
5328e1d resolves this TODO outright; I'll changes the other comments shortly.

Reply:
Turns out that `{{/* a comment */}}` unfortunately doesn't work for me. I suspect it might be related to these being markdown sources, though it would be an odd bug if the comment stripper happened _after_ the markdown conversion.

Reply:
Turns out the actual reason for this not working is that Hugo doesnt run Go's templating engine over the content files, so never strips out the comments -_- D'oh!